code/Architecture.md

```diff
@@ -0,0 +1,47 @@
+# Architecture
+
+This document describes how things are implemented.
```

Comment:
could this sentence be a little more descriptive?  what "things"?  Maybe outline the purpose of this document?